web/docs/developer-guides/build-from-source.md

```diff
@@ -34,7 +34,7 @@ Every [release](https://github.com/tenzir/tenzir/releases) of Tenzir includes an
 |✓|[OpenSSL](https://www.openssl.org)||Utilities for secure networking and cryptography.|
 |✓|[FlatBuffers](https://google.github.io/flatbuffers/)|>= 1.12.0|Memory-efficient cross-platform serialization library.|
 |✓|[Boost](https://www.boost.org)|>= 1.81.0|Required as a general utility library.|
-|✓|[Apache Arrow](https://arrow.apache.org)|>= 12.0.0|Required for in-memory data representation. Must be built with Compute, Zstd and Parquet enabled.|
+|✓|[Apache Arrow](https://arrow.apache.org)|>= 13.0.0|Required for in-memory data representation. Must be built with Compute, Zstd and Parquet enabled. For the `s3` plugin, Filesystem also needs to be enabled.|
```

Comment:
This needs to also be adjusted in CMake:

https://github.com/tenzir/tenzir/blob/afd5c18d757cc6783f063547ace736bbc74a9429/libtenzir/CMakeLists.txt#L65

Reply:
Adjusted now, I hope.